Dockerfile

```
@@ -1,10 +1,11 @@
-FROM opendatacube/geobase:wheels-3.0.4 as env_builder
 ARG py_env_path=/env
+ARG V_BASE=3.3.0
 
+FROM opendatacube/geobase-builder:${V_BASE} as env_builder
 ENV LC_ALL=C.UTF-8
 
 # Install our Python requirements
-COPY docker/requirements.txt docker/version.txt /conf/
+COPY docker/requirements.txt docker/version.txt docker/constraints.txt /conf/
 
 RUN cat /conf/version.txt && \
   env-build-tool new /conf/requirements.txt ${py_env_path}
```

I think you need to add constraints here.
```suggestion
  env-build-tool new /conf/requirements.txt /conf/constraints.txt ${py_env_path}
```